ui/packages/ui/src/app/pages/connectors/ConnectorsTableComponent.tsx

```diff
@@ -183,18 +192,24 @@ export const ConnectorsTableComponent: React.FunctionComponent<IConnectorsTableC
       });
   }
   
-  const getTaskStates = (conn: Connector) => {
+  const getTaskStates = (conn: Connector, connName, restartConfirmation) => {
     const taskElements: any = [];
     const statesMap = new Map(Object.entries(conn.taskStates));
+
     statesMap.forEach((taskState: any, id: string) => {
       taskElements.push(
         <ConnectorTask
           key={id}
           status={taskState.taskStatus}
+          connName={conn.name}
           taskId={id}
           errors={taskState.errors}
           i18nTask={t('task')}
+          i18nRestart={t('restart')}
           i18nTaskStatusDetail={t('taskStatusDetail')}
+          i18nTaskErrorTitle={t('taskErrorTitle')}
+          setConnectorToRestart={connName}
```

Comment:
should be 'setConnectorTaskToRestart', which would initiate restart of the designated task number for the connector